packages/expo/package.json

```diff
@@ -49,11 +49,11 @@
     "bugsnag-build-reporter": "^1.0.1"
   },
   "devDependencies": {
-    "expo-constants": "~13.2.3"
+    "expo-constants": "~14.0.2"
   },
   "peerDependencies": {
-    "expo": ">=33.0.0",
-    "expo-constants": "~13.2.3",
+    "expo": "~47.0.3",
```

Comment:
only thing I'm unsure about here is the required version of expo

Reply:
I think either leave it as it was (not sure why it was left at `>=33.0.0` previously — maybe just to reduce the number of changes necessary for a new version?) or change it to `^47.0.0`